api/opencl_runtime_layer.asciidoc

```diff
@@ -5427,6 +5427,23 @@ handle is used by an OpenCL command queued to a command-queue without being
 acquired.
 This is to guarantee that the state of the memory objects is up-to-date and
 they are accessible to OpenCL.
+
+The following restrictions shall apply -
+  * Each memory object must be acquired only once. Acquiring memory object 
+    multiple times without releasing it may result in implementation defined
+    behavior.
+  * The acquire must be performed on a command-queue associated with a device
+    that was one of the devices specified via {CL_MEM_DEVICE_HANDLE_LIST_KHR}
+    when the memory object was imported using {clCreateBufferWithProperties} or
+    {clCreateImageWithProperties}. If {CL_MEM_DEVICE_HANDLE_LIST_KHR} was not
+    specified, the acquire can be performed on a command-queue associated with
+    any device in the context.
+  * The memory object will be acquired on all devices specified
```

Comment:
Very minor point but I think I'd suggest "for all devices" instead of "on all devices". I think it's better to describe this in terms of the outcome of the command rather than suggest that there might be an operation performed for each device if that makes sense.